src/run_script.js

```diff
@@ -1,10 +1,9 @@
 const { dirname } = require('path')
 
-const pathExists = require('path-exists')
-
 // Retrieve the command to run `package.json` `scripts` commands
-const getRunScriptCommand = async function ({ projectDir, packageJsonPath = projectDir }) {
-  if (await pathExists(`${dirname(packageJsonPath)}/yarn.lock`)) {
+const getRunScriptCommand = async function ({ locatePath, packageJsonPath }) {
+  const exists = (await locatePath([`${dirname(packageJsonPath)}/yarn.lock`])) !== undefined
```

Comment:
What's the difference from using `path-exists`? 
(except that this returns `false` when `yarn.lock` is a directory, but that seems unlikely)

Reply:
`path-exists` is a Node dependency which I wanted to remove. We can keep it and add another dependency to `context`

Reply:
Isn't `locate-path` a Node dependency as well?

Reply:
Oh, I am understanding now. You're unifying both `path-exists` and `locate-path` into a single function, so it is easier to adapt for non-Node.js environments.
Sounds good :+1: